spec/integration/init_web_spec.cr

```diff
@@ -28,7 +28,7 @@ describe "Initializing a new web project" do
   it "creates a full web app with generator" do
     puts "Web app generators: Running integration spec. This might take awhile...".colorize(:yellow)
     with_project_cleanup do
```

Comment:
This line drops the DB after it's done. However, since we never ran the setup in this spec, no DB was ever created. This should have always failed since it was added in November 2020, but if you look at the last few merged PRs, you'll see it's been passing....  (👻 )